assets/javascripts/discourse/connectors/bread-crumbs-right/solved-status-filter.js.es6

```diff
@@ -3,7 +3,12 @@ import DiscourseUrl from "discourse/lib/url";
 
 export default {
   shouldRender(args, component) {
-    if (!component.siteSettings.show_filter_by_solved_status) {
+    const router = Discourse.__container__.lookup("router:main");
```

Comment:
Can we do `component.container` here? `Discourse.` is deprecated.

Reply:
Sure, no problem. Probably ideal to update the existing `Discourse.` use as well?

https://github.com/discourse/discourse-solved/blob/913974e620553409334bacac12be87511c6a2d72/assets/javascripts/discourse/connectors/bread-crumbs-right/solved-status-filter.js.es6#L11-L13

Reply:
There was no direct `component.container` available, but I was able to find something that seems to work!

Reply:
`register` is another private API unfortuantely. In that case could you use our `getOwner` helper? It can get a container.

Reply:
Ah ha! Thanks for the `getOwner` tip. I hadn't used/noticed it used before.